teeio-validator/library/pcie_ide_test_lib/test_case/ide_km_common.c

```diff
@@ -30,6 +33,8 @@ const char *direction_names[] = {
 const char *substream_names[] = {
     "PR", "NPR", "CPL"};
 
+static uint8_t g_pcie_ide_fixed_key = TEEIO_KEY_GEN_FIXED_KEY;
```

Comment:
Please dont do that. because the value will be changed, not fixed. it is hard to debug.

Just define MACRO and make it really fixed.

Reply:
Will update.